tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -25,7 +25,7 @@ describe('Acceptance: brocfile-smoke-test', function() {
   this.timeout(500000);
 
   before(function() {
-    return createTestTargets(appName);
+    return createTestTargets(appName, { createESLintConfig: true });
```

Comment:
this option does not actually run the blueprint, it just generates a very basic `.eslintrc.js` file